Developer-Guide.md

```diff
@@ -265,10 +265,13 @@ As a prerequisite, you need to install `libelf-dev` and `bc`. Otherwise, you
 will not be able to build the kernel from sources.
 
 ```
-$ kernel_arch="$(arch)"
+$ go get github.com/kata-containers/tests
+$ cd $GOPATH/src/github.com/kata-containers/tests/.ci
+$ kernel_arch="$(./kata-arch.sh --golang)"
+$ kernel_dir="$(./kata-arch.sh --kernel)"
```

Comment:
\o/ :smile:

Reply:
@jodh-intel : Thanks for your work regarding the script. I have fixed my patch based on your above comment.